File: cpp/bioskop.cpp
#include<bits/stdc++.h>
using namespace std;

class Studio
{
    // mendefinisikan private atribut
    private:
    string kode;
    string lokasi;
    string jaringan;
    int kapasitas;

    // mendefinisikan public constructor, method, dan destructor
    public:
    Studio(){}; // constructor kosong

    // constructor dengan parameter
    Studio(string kode, string lokasi, string jaringan, int kapasitas)
    {
        this->kode = kode;
        this->lokasi = lokasi;
        this->jaringan = jaringan;
        this->kapasitas = max(kapasitas,5);
    };

    // setter method
    void setKode(string kode)
    {
        this->kode = kode;
    }
    void setLokasi(string lokasi)
    {
        this->lokasi = lokasi;
    }
    void setJaringan(string jaringan)
    {
        this->jaringan = jaringan;
    }
    void setKapasitas(int kapasitas)
    {
        this->kapasitas = max(kapasitas,5);
    }

    // getter method
    string getKode()
    {
        return kode;
    }
    string getLokasi()
    {
        return lokasi;
    }
    string getJaringan()
    {
        return jaringan;
    }
    int getKapasitas()
    {
        return kapasitas;
    }

    // other method

    // destructor
    ~Studio(){};
};

File: python/bioskop.py
class Studio:
    __kode = ""
    __lokasi = ""
    __jaringan = ""
    __kapasitas = 0

    def __init__(self, kode:str, lokasi:str, jaringan:str, kapasitas:int):
        self.__kode = kode
        self.__lokasi = lokasi
        self.__jaringan = jaringan
        self.__kapasitas = kapasitas

    def getKode(self):
        return self.__kode

    def getLokasi(self):
        return self.__lokasi

    def getJaringan(self):
        return self.__jaringan

    def getKapasitas(self):
        return self.__kapasitas

    def setKode(self, kode:str):
        self.__kode = kode

    def setLokasi(self, lokasi:str):
        self.__lokasi = lokasi

    def setJaringan(self, jaringan:str):
        self.__jaringan = jaringan

    def setKapasitas(self, kapasitas:int):
        self.__kapasitas = kapasitas